cmake/addExecutable.cmake

```diff
@@ -35,4 +35,23 @@ macro(alpaka_add_executable In_Name)
             set_property(TARGET ${In_Name} PROPERTY CUDA_SEPARABLE_COMPILATION OFF)
         endif()
     endif()
+
+    if(alpaka_ACC_GPU_HIP_ENABLE)
+        enable_language(HIP)
+        foreach(_file ${ARGN})
+            if((${_file} MATCHES "\\.cpp$") OR
+               (${_file} MATCHES "\\.cxx$") OR
+               (${_file} MATCHES "\\.cu$") OR
```

Comment:
same here, probably not cu

Reply:
Thanks for the hint. Looks like a copy paste error. I will overtake the PR from @j-stephan  since he is not available for the next time. Let's see, if I find some time this week.